docs/configuration_public.md

```diff
@@ -35,7 +35,7 @@ spec:
         keyName: app-key
 {{< /highlight >}}
 
-{{% collapse-content title="Parameters" level="h4" expanded=true id="global-options-list" %}}
+{{%% collapse-content title="Parameters" level="h4" expanded=true id="global-options-list" %%}}
```

Comment:
Not sure about this one, @DataDog/documentation could you confirm whether or not this change makes sense and same below ?